packages/element-templates-json-schema-shared/src/defs/examples.json

```
@@ -56,6 +56,20 @@
         "type": "simple",
         "property": "httpMethod",
         "oneOf": [ "POST", "PUT", "DELETE" ]
+      },
+      {
+        "allMatch": [
+          {
+            "type": "simple",
+            "property": "httpMethod",
+            "equals": "GET"
+          },
+          {
+            "type": "simple",
+            "property": "httpMethod",
+            "oneOf": [ "POST", "PUT", "DELETE" ]
+          }
+        ]
```

Let's change the example as the condition is impossible to be met.

```suggestion
        "allMatch": [
          {
            "type": "simple",
            "property": "authType",
            "equals": "Basic"
          },
          {
            "type": "simple",
            "property": "httpMethod",
            "oneOf": [ "POST", "PUT", "DELETE" ]
          }
        ]
```